spec/rails_helper.rb

```diff
@@ -174,9 +174,11 @@ def self.capybara_tmp_path
     Faker::UniqueGenerator.clear # Clears used values to avoid retry limit exceeded error
   end
 
+  # rubocop:disable Rails/RedundantTravelBack
```

Comment:
In theory we shouldn't need this line as Rails supposedly since 5.2 automatically runs `travel_back` after every test.

In practice, lots of tests fail due to time related issues if we remove these lines,